apps/api/src/integration/plugins/discord.integration.ts

```diff
@@ -54,11 +54,36 @@ export class DiscordIntegration extends BaseIntegration {
     return new Set(['webhookUrl'])
   }
 
-  public init(): Promise<void> {
-    // TODO: Send a text message to the discord channel confirming keyshade was added
-    return Promise.resolve()
+// Edited Code starts for init 
+  public async init(): Promise<void> {
+  const integration = this.getIntegration<DiscordIntegrationMetadata>()
```

Comment:
This should be also creating an integration run. refer https://github.com/keyshade-xyz/keyshade/blob/fbdfb2740a6178eb8031b1f7d158fddc5a9c7b1a/apps/api/src/integration/plugins/discord.integration.ts#L92